ilastik/shell/gui/ilastikShell.py

```diff
@@ -632,14 +635,92 @@ def _loaduifile(self):
         styleStartScreenButton(self.startscreen.browseFilesButton, ilastikIcons.OpenFolder)
         self.startscreen.browseFilesButton.clicked.connect(self.onOpenProjectActionTriggered)
 
-        pos = 1
-        for workflow, _name, displayName in getAvailableWorkflows():
-            b = QToolButton(self.startscreen, objectName="NewProjectButton_" + workflow.__name__)
-            styleStartScreenButton(b, ilastikIcons.GoNext)
-            b.setText(displayName)
-            b.clicked.connect(partial(self.loadWorkflow, workflow))
-            self.startscreen.VL1.insertWidget(pos, b)
-            pos += 1
+        self._populateWorkflows()
+
+    def _populateWorkflows(self):
+        wf_startup_menu_order = {
+            "Segmentation Workflows": {
+                "workflows": [
+                    "Pixel Classification",
+                    "AutocontextTwoStage",
+                    "Neural Network Classification (Local)",
+                    "Neural Network Classification (Remote)",
+                    "Carving",
+                    "Edge Training With Multicut",
+                ],
+                "init_expanded": True,
+            },
+            "Object Classification Workflows": {
+                "workflows": [
+                    "Object Classification (from prediction image)",
+                    "Object Classification (from binary image)",
+                ],
+                "init_expanded": True,
+            },
+            "Tracking Workflows": {
+                "workflows": [
+                    "Manual Tracking Workflow",
+                    "Automatic Tracking Workflow (Conservation Tracking) from binary image",
+                    "Automatic Tracking Workflow (Conservation Tracking) from prediction image",
+                    "Animal Conservation Tracking Workflow from Binary Image",
+                    "Animal Conservation Tracking Workflow from Prediction Image",
+                    "Structured Learning Tracking Workflow from binary image",
+                    "Structured Learning Tracking Workflow from prediction image",
+                ],
+                "init_expanded": False,
```

Comment:
Since we renamed the `init_expanded` parameter, we might as well rename keys here too.